exporter/prometheusremotewriteexporter/README.md

```diff
@@ -43,6 +43,10 @@ Example:
 exporters:
   prometheusremotewrite:
     endpoint: "http://some.url:9411/api/prom/push"
+    wal:
+        path: "DIRECTORY_HERE"
+        truncate-frequency: "Optional Frequency in seconds per time.ParseDuration https://golang.org/pkg/time/#ParseDuration, default of 1m"
+        cache-size: "An optional size before truncating the WAL, default of 300"
```

Comment:
Same here. How to choose it?

Reply:
Sure, I've updated the comments about what cache_size is. Done, thanks!